pages/k8s/1.34/release-notes.md

```diff
@@ -0,0 +1,94 @@
+
+---
+wrapper_template: templates/docs/markdown.html
+markdown_includes:
+  nav: kubernetes/charmed-k8s/docs/shared/_side-navigation.md
+context:
+  title: 1.34 Release notes
+  description: Release notes for Charmed Kubernetes
+keywords: kubernetes, release, notes
+tags:
+  - news
+sidebar: k8smain-sidebar
+permalink: 1.34/release-notes.html
+layout:
+  - base
+  - ubuntu-com
+toc: False
+
+---
+
+# 1.34
+
+### September 24, 2025 - `charmed-kubernetes --channel 1.34/stable`
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.34/bundle.yaml).
+
+## What's new
+### aws-integrator
+[LP#2111261] Tag iam roles and policies with juju-model-uuid (#8)
+Pin dependencies to 1.34 branch
+### kubernetes-control-plane
+fix: support dual stack kubelet node ip addresses
+### kubernetes-e2e
+Cluster Validation using Canonical K8s (#35)
```

Comment:
I am not really understanding this entry. This is a shared charm with Canonical K8s and there was work in it this release?

Reply:
that's right.  This charm `kubernetes-e2e` now works with Canonical K8s and Charmed Kubernetes.  This is one of the charms we release with all the charmed kubernetes charms, but it now supports Canonical K8s as well as `kubernetes-control-plane` and `kubernetes-worker`